recipes/jenkins.rb

```diff
@@ -112,6 +112,24 @@
   end
 end
 
+directory '/var/lib/jenkins/init.groovy.d' do
+  owner 'jenkins'
+  group 'jenkins'
+end
+
+file '/var/lib/jenkins/init.groovy.d/start-quiet.groovy' do
```

Comment:
I proposed using number prefixes to prevent scripts dependency errors:

See https://github.com/ros-infrastructure/cookbook-ros-buildfarm/pull/147.

```suggestion
file '/var/lib/jenkins/init.groovy.d/03-start-quiet.groovy' do
```